explainability-arrow/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <parent>
+    <groupId>org.kie.trustyai</groupId>
+    <artifactId>trustyaiexplainability</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>explainability-arrow</artifactId>
+  <version>2.0.0-SNAPSHOT</version>
+  <name>TrustyAI Arrow</name>
+
+  <properties>
+    <version.org.apache.arrow>7.0.0</version.org.apache.arrow>
+    <version.io.netty>4.1.68.Final</version.io.netty>
+    <version.com.google.flatbuffers>2.0.3</version.com.google.flatbuffers>
+  </properties>
```

Comment:
Usually I prefer to declare everything in the parent pom but in this case I'm fine with this because Apache Arrow and Netty are quite specific dependencies so I'm fine to keep them here